mobly/controllers/android_device.py

```diff
@@ -648,13 +649,14 @@ def _terminate_sl4a(self):
         Send terminate signal to sl4a server; stop dispatcher associated with
         the session. Clear corresponding droids and dispatchers from cache.
         """
+        if self.ed:
+            self.ed.clean_up()
+            self.ed = None
         if self.sl4a:
             self.sl4a.closeSl4aSession()
```

Comment:
I believe it's here, right?
https://android.googlesource.com/platform/external/sl4a/+/master/Common/src/com/googlecode/android_scripting/jsonrpc/JsonRpcServer.java#102